### PR TITLE
Fix hyper cube with cylindrical hole

### PR DESCRIPTION
**Pull request by drwells**
This is part of #6274: we need to clear manifolds at the start to avoid wobbling outside edges.

### CODE REVIEW

**Comment by drwells** [comment]
To be precise: I noticed this while working on updating step-49.

File: tests/grid/grid_out_svg_02.cc [hunk 1]
```diff
@@ -22,6 +22,7 @@
 #include <deal.II/grid/manifold_lib.h>
 #include <deal.II/grid/grid_out.h>
 
+// Check EPS output with a GridGenerator function.
```
Comment (bangerth) [inline tests/grid/grid_out_svg_02.cc]: I think you want to say "SVG output"

**Comment by kronbichler** [comment]
@drwells Can you please look into the test `tria_boundary_id_01` that now fails due to the change in the manifold id?

**Comment by drwells** [comment]
yes; I am rerunning the test suite now.

**Comment by tamiko** [comment]
@drwells I would prefer to just fix this one failing test we spotted. I will trigger `tester` right after this got merged.

**Comment by drwells** [comment]
My workstation is almost done running everything. I only offered because I have already compiled all of the test programs :)

Anyway: the change in this test is expected because the manifold ids should be set to flat_manifold_id everywhere except the interior (which is exactly the difference in the output). I'll upload another commit in a moment.

**Comment by drwells** [comment]
Okay, that should fix the only other test failure.

**Pull request merged**